proto/info.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+option optimize_for = LITE_RUNTIME;
+package valhalla;
+
+message Statistic {
+  optional string name = 1;  // the name of the statistic
+  optional double value = 2; // the value of the statistic
```

Comment:
with proto3 we could use OneOf here which might be a good idea (just add all the primative types). any thoughts?

Reply:
Devils advocate: Is there a need for that?

Most usecases can make do with a floating point value. Do you have any examples that need something else?

Reply:
yeah i totally think in the context of metrics it makes sense to just have a single floating point value, i was just trying to suggest that maybe i didnt think of all the possible use-cases and that if there were more we could fix it with OneOf. currently i still cannot come up with a good use-case for anything but numbers though :smile:

Reply:
i could kind of see a metric where an enum might make sense, in which case maybe strings would be better than numbers :man_shrugging:

Reply:
How is this proro msg intended to be consumed/used? Do you envision users using it to emit metrics to an external metrics collector (eg `statsd`/`prometheus` etc)?

If these statistics have to be emitted as metrics, then the user might also want to know what type of stat is within. Common types would be counters, gauges & timer (they all have different meaning for a metrics collector and hence have to be distinguished before being sent). Or is it intended to only have timing information?

We could also emit the metrics directly from the place of measurement, in which case we'd know what is it that we're measuring (timer/counts etc).

Reply:
i didnt add it to this pr but what i want to do is refactor serialization to add this info to the serialized response (when a preprocessor define is turned on)